### ISSUE TITLE
Do we want to change to the new tab script?

### ISSUE THREAD

**Issue by astridx** [joomla-projects/gsoc18_override_management#43]
Do we want to change to the new tab script?
https://github.com/joomla/joomla-cms/issues/21142

**Comment by Anu1601CS** [comment]
Yes, of course. But, we are getting some issue in the new Joomla tab function.

**Comment by dgrammatiko** [comment]
Please share your html code for the tabs (ie chrome->source). I suspect that you're missing one or more ids

**Comment by Anu1601CS** [comment]
@dgrammatiko Thanks, You can test this branch and file.
https://github.com/joomla-projects/gsoc18_override_management/pull/47/files#diff-e7af1d7b5598630a5b8aee6275f462b7

Changes occurs in 
`administrator/components/com_templates/tmpl/template/default.php`

Visit: `administrator/index.php?option=com_templates&view=template&id=510&file=aG9tZQ==`

In first reload its working fine. 

![screenshot from 2018-07-18 18-28-31](https://user-images.githubusercontent.com/30978328/42883815-491ca43a-8aba-11e8-88d8-ab4aae068983.png)

But, if I click on the second tab and just reload this happens
![screenshot from 2018-07-18 18-28-58](https://user-images.githubusercontent.com/30978328/42883861-6a7a1950-8aba-11e8-9fca-cb2ce9ed8bd0.png)

Chrome->source
![screenshot from 2018-07-18 18-33-31](https://user-images.githubusercontent.com/30978328/42883927-912e5f0c-8aba-11e8-8b92-9b6537a4c15e.png)

**Comment by dgrammatiko** [comment]
empty your local storage

**Comment by Anu1601CS** [comment]
> empty your local storage

**Comment by astridx** [comment]
See https://github.com/joomla/joomla-cms/pull/21183

**Comment by dgrammatiko** [comment]
@astridx @Anu1601CS can I have a test: 
https://github.com/joomla/joomla-cms/pull/21204
